build-java/examples/src/main/java/examples/ExampleApplication.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package examples;
+
+import com.criteo.marketing.ApiClient;
+import com.criteo.marketing.ApiException;
+import com.criteo.marketing.Configuration;
+import com.criteo.marketing.api.AdvertisersApi;
+import com.criteo.marketing.model.CampaignMessage;
+
+import java.util.List;
+
+public class ExampleApplication {
+    public static void main(String[] args) throws ApiException {
+        ApiClient client = Configuration.getDefaultApiClient();
+        client.setUsername("client-id");
+        client.setPassword("client-secret");
```

Comment:
You can add an example line (commented) to show how to enable debugging in logs?

Reply:
done